src/components/ble/NimbleController.h

```diff
@@ -42,18 +42,19 @@ namespace Pinetime {
 
     public:
       NimbleController(Pinetime::System::SystemTask& systemTask,
-                       Pinetime::Controllers::Ble& bleController,
+                       Ble& bleController,
                        DateTime& dateTimeController,
-                       Pinetime::Controllers::NotificationManager& notificationManager,
-                       Controllers::Battery& batteryController,
+                       NotificationManager& notificationManager,
+                       Battery& batteryController,
                        Pinetime::Drivers::SpiNorFlash& spiNorFlash,
-                       Controllers::HeartRateController& heartRateController,
-                       Controllers::MotionController& motionController,
-                       Pinetime::Controllers::FS& fs);
+                       HeartRateController& heartRateController,
+                       MotionController& motionController,
+                       FS& fs);
       void Init();
       void StartAdvertising();
       int OnGAPEvent(ble_gap_event* event);
 
+      /* these are not implemented yet
```

Comment:
You can safely remove those declarations of the methods are not implemented in NimbleController.cpp.

Reply:
I didn't remove them because I wasn't sure if they had any value from a documentation standpoint.

Reply:
I think I forgot to remove them a long time ago. They are not needed since these functionalities are implemented is services.